.github/workflows/build_and_test.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       max-parallel: 5
       matrix:
-        python-version: ["3.8"]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
So I added python "3.9", "3.10", "3.11" so that we are certain that we are not introducing some incompatibility somewhere